scripts/test-infra/build-image-cross.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash -e
 
 # cross build
-make image-all
+IMAGE_ALL_PLATFORMS=linux/amd64,linux/arm64 make image-all
```

Comment:
@marquiz WDYT

Reply:
PING @marquiz

Reply:
Maybe this wouldwork for now. We don't check buildability of armv7 images on every PR. We'd see build failures after the fact. But arch-specific build failures have been super rare so we can probably bear this for now.